tests/unit/test_charm.py

```diff
@@ -40,47 +38,43 @@
 CREATE_CLUSTER_CONF_PATH = "/etc/postgresql-common/createcluster.d/pgcharm.conf"
 
 
-# @pytest.mark.usefixtures("juju_has_secrets")
-class TestCharm(unittest.TestCase):
-    def setUp(self):
-        self._peer_relation = PEER
-        self._postgresql_container = "postgresql"
-
-        self.harness = Harness(PostgresqlOperatorCharm)
-        self.addCleanup(self.harness.cleanup)
-        self.harness.begin()
-        self.charm = self.harness.charm
-        self.rel_id = self.harness.add_relation(self._peer_relation, self.charm.app.name)
-        self.harness.add_relation("upgrade", self.charm.app.name)
-
-    @pytest.fixture
-    def use_caplog(self, caplog):
-        self._caplog = caplog
-
-    @patch_network_get(private_address="1.1.1.1")
-    @patch("charm.subprocess.check_call")
-    @patch("charm.snap.SnapCache")
-    @patch("charm.PostgresqlOperatorCharm._install_snap_packages")
-    @patch("charm.PostgresqlOperatorCharm._reboot_on_detached_storage")
```

Comment:
Why move away from patch decorators?

Reply:
My main motivation here was to make clear what parameter correspond to what patch (also not needing to have blank params). It was also the approach adopted in Kafka's repo I think. I don't have a strong opinion here, can revert if wanted.

Another option would be to convert the patches into fixtures, but this would considerably increase verbosity in the beginning of the file.

Reply:
OK, I'm fine with the change.

Reply:
I'm also fine with the change. Let's use the patches as they were implemented in this PR.